erpnext/accounts/utils.py

```
@@ -754,10 +752,7 @@ def update_reference_in_payment_entry(
 		existing_row = payment_entry.get("references", {"name": d["voucher_detail_no"]})[0]
 
 		# Update Advance Paid in SO/PO since they are getting unlinked
-		advance_payment_doctypes = frappe.get_hooks("advance_payment_receivable_doctypes") + frappe.get_hooks(
-			"advance_payment_payable_doctypes"
-		)
-		if existing_row.get("reference_doctype") in advance_payment_doctypes:
+		if existing_row.get("reference_doctype") in get_advance_payment_doctypes:
```

Pylint (3.3.7)

[error] Value 'get_advance_payment_doctypes' doesn't support membership test

(E1135)